docs/man/docker-inspect.1.md

```diff
@@ -158,8 +158,13 @@ To get information on a container use its ID or instance name:
             "Running": true,
             "StartedAt": "2014-04-04T21:33:54.16259207Z",
         },
-        "Volumes": {},
-        "VolumesRW": {},
+        "MountPoints": {
+          "/data": {
+            "Source": "/data",
+            "Destination": "/data",
+            "RW" false
```

Comment:
There were some other formatting issues in the docs as well, but better hold that until we're in "docs review", otherwise it gets confusing. :)